tests/e2e-playwright/Makefile

```diff
@@ -137,6 +137,35 @@ test-sleepers-anywhere: _check_venv_active $(USER_INPUT_FILE) ## test sleepers a
 		--browser chromium \
 		--headed
 
+# Define the file where user input will be saved
+JUPYTER_USER_INPUT_FILE := .e2e-playwright-jupyterlab-env.txt
+
+# Prompt the user for input and store it into variables
```

Comment:
Q: why not inline doc as `target: ## Prompt to ...

Reply:
That is some chatty stuff

Reply:
this command looks nice!

Reply:
ok I moved it inline, but I am not sure of the effect. since this starts with a dot it will not show in the makefile help